polling_stations/apps/councils/models.py

```diff
@@ -140,6 +142,36 @@ def import_script_path(self):
 
         return str(import_script_path)
 
+    @property
+    def has_polling_stations_in_db(self):
+        if self.pollingstation_set.count() > 0:
+            return True
+        return False
+
+    def latest_data_event(self, event_type):
+        data_event_model = apps.get_model("data_importers", "DataEvent")
```

Comment:
That's helpful, as I hadn't thought about the string option. 
There is this line: https://github.com/DemocracyClub/UK-Polling-Stations/blob/c426b89f5eb48827900fa6cc546778bd8000db98/polling_stations/apps/data_importers/models.py#L28 in the `data_importers/models.py` module, which would still need the `apps.get_model` to avoid the circular import issue.

Reply:
But is maybe a better place for the complexity to live.

Reply:
Just went on a bit of tangent trying to override the `Council.save` method as:
 
```
def save(self, **kwargs):
    super().save()
    DataQuality.objects.get_or_create(council_id=self.council_id)
```

This didn't work, when creating council records using the `CouncilFactory`.

Not worth getting my head around it. So just going to shift `apps.get_model` to the `data_importers/models.py` and leave it for now. Getting rid of the signal is a nice to have, but not needed.

Reply:
```
def council_saved(sender, **kwargs):
    DataQuality.objects.get_or_create(council=kwargs["instance"])


council_model = apps.get_model("councils", "Council")
post_save.connect(council_saved, sender=council_model)
```

Doesn't work either because `raise AppRegistryNotReady("Models aren't loaded yet.")`.

I think I'm going to sack it off as Too Difficult and leave it as is.